data_management/hkh_watermaps.json

```
@@ -6,8 +6,8 @@
     "processingLevel": ["SLC"],
     "beamMode": ["IW"],
     "polarization": ["VV+VH"],
-    "start": "2022-05-18",
-    "end": "2022-12-01",
+    "start": "2023-01-17",
```

```suggestion
    "start": "2023-01-01",
```
per our discussion in scrum